integration_tests/grid_fee.feature

```diff
@@ -83,7 +83,7 @@ Feature: GridFee integration tests
   Examples: Minimum Offer Age
      | min_offer_age | slot_length | tick_length |
      |       0       |     60      |     60      |
-     |       5       |     60      |     15      |
+     |       5       |     60      |     10      |
```

Comment:
I assume that this is not necessary and only a remnant of testing, right ?

Reply:
Resolved after offline discussion,

> It kind of is necessary, as the more you increase the delay in offer/bid propagation while keeping tick_length constant, it would be delay propagation to each and every corner, the reason this test was passing before is due to multiple increment of current_tick, can also see the correlation one line below your comment where when  tick_length  is decreased even further to 5 when min_offer_age was increased as high as 9.